src/components/ChangelogSnippet/ChangelogCard.astro

```diff
@@ -22,14 +24,14 @@ const formatDate = (dateString: string) => {
 const formattedDate = formatDate(item["date-of-release"])
 ---
 
-<div class={styles.cardWrapper} data-expandable="wrapper">
+<div class={clsx(styles.cardWrapper, showBorder && styles.withBorder)} data-expandable="wrapper">
   <div class={styles.card} data-expandable="card">
     <div class={styles.header}>
       <div class={styles.metaSection}>
```

Comment:
This is missing extra info:
<img width="306" height="103" alt="image" src="https://github.com/user-attachments/assets/c13f8aa5-304d-4e35-9902-dbe0fa730e44" />

Do you need them in the algolia Index?